### PR TITLE
add changelog 

### PR DESCRIPTION
**Pull request by jasonnovichRunAI**
changelog added from 2.13

### CODE REVIEW

File: docs/home/changelog/hotfixes-2-15.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+## Version 2.15.4 - January 5, 2024
+
+### Fixed issues
+
+| Internal ID | Description |
+|--|--|
+| RUN-15026 | Fixed an issue where building workloads to send to the cluster were not built inside volumes for backwards compatibility. |
+| RUN-14907 | Fixed an issue where after an upgrade, the Analytics Dashboard is missing the time ranges from before the upgrade.|
+| RUN-14903 | Fixed an issue where the project status is audited creating long logs. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-15.md:27]: I would remove it. Does not seem customer facing + KCL issues are mapped usually by our team

File: docs/home/changelog/hotfixes-2-15.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+| RUN-14062 | Fixed an issue where the Running Workload per Type Graph box content is too big for the frame. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-15.md:28]: Where? In the overview dashboard. Missing the context

File: docs/home/changelog/hotfixes-2-15.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+## Version 2.15.2 - February 5, 2024
+
+### Fixed issues
+
+| Internal ID | Description |
+|--|--|
+| RUN-14645 | Fixed an issue where a workload volume is mounted as read-only when created in the workspace training form. |
+| RUN-14635 | Fixed an issue where the Status updater was missing the correct permissions. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-15.md:37]: That's the solution, which customers care less about if at all. What did this issue cause? You should ask Roman, as there is no info in the bug.

File: docs/home/changelog/hotfixes-2-15.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+| RUN-14472 | Fixed an issue where template updates were not being applied to the workload. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-15.md:38]: Not clear which template and what was the impact on the user.

File: docs/home/changelog/hotfixes-2-15.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+| RUN-14434 | Fixed an issue where `runai_allocated_gpu_count_per_gpu` was multiplied by seven. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-15.md:39]: Seems like it dud not happen all the time. When does it happen? The details of the inaccuracy are less relevant. What does runai_allocated_gpu_count_per_gpu mean for a customer?

File: docs/home/changelog/hotfixes-2-15.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+| RUN-14425 | Fixed an issue when calculating deserved quota metrics for project, nodepool, or cluster causing unlimited quotas to display -1. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-15.md:40]: So the symptom is that an unlimited project quota is displayed like -1 in some cases. What are the cases? I did not understand the relevance on mentioning nodepool & cluster

File: docs/home/changelog/hotfixes-2-15.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+| RUN-13401 | Fixed an issue where there was no accurate error information when a predicate fails (was `not found`). |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-15.md:41]: what can a customer do with this info? Did you check if we have the term predicate documented??

**Review by omerbenedictrunai — COMMENTED** [review]
I gave up at some point. The 2.15 bugs are not ready for review.

**Comment by jasonnovichRunAI** [comment]
@omerbenedictrunai @yarongol is ready for review.

File: docs/home/changelog/hotfixes-2-13.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+
+| Internal ID | Description  |
+| ---------------------------- | ---- |
+| RUN-11367 | Fixed an issue where a double click on *SSO Users* redirects to a blank screen. |
+
+## Version 2.13.25
+
+| Internal ID | Description  |
+| ---------------------------- | ---- |
+| RUN-13171 | Fixed an issue when a cluster is not connected the actions in the *Workspace* and *Training* p[ages are still enabled. After the corrections, the actions will be disabled. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-13.md:49]: P[ages? must be a typo

File: docs/home/changelog/hotfixes-2-13.md [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+* <!-- RUN-12405 Refine out of the box alerts to run:ai rules --> Added and refined alerts to the state of Run:ai components, schedule latency, and warnings for out of memory on Jobs.
+* <!-- RUN-11711 Pod Security Admission issues in control plane -->Added the ability to work with restricted PSA policy.
+
+### Fixed issues
+
+| Internal ID | Description  |
+|-------------- | ------------------- |
+| RUN-12650 | Fixed an issue that used an incorrect metric in analytics GPU ALLOCATION PER NODE panel. Now the correct allocation is in percentage. |
+| RUN-12602 | Fixed an issue in `runaiconfig` where the `WorkloadServices` spec has memory requests/limits and cpu requests/limits and gets overwritten with the system default. |
+| RUN-12587 | Fixed an issue where elastic scheduling of jobs enters an endless loop. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-13.md:90]: please remove this bug

File: docs/home/changelog/hotfixes-2-15.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+
+* <!-- RUN-14077 - [runai-cli] allow configuring client burst and QPS -->Added environment variables for customizable QPS and burst support.
+
+* <!-- RUN-13968 - Prometheus high availability -- allow changing # of replicas -->Added the ability to support running multiple Prometheus replicas.
+
+### Fixed issues
+
+| Internal ID | Description |
+|--|--|
+| RUN-14292 | Fixed an issue where Bright installations were failing due to missing `create cluster` permissions. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-15.md:50]: Bright needs to be replaced with BCM

File: docs/home/changelog/hotfixes-2-15.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+| RUN-14289 | Fixed an issue where metrics were not working due to an incorrect parameter in the cluster-config file. |
+| RUN-14198 | Fixed an issue in services where multi nodepool jobs were not scheduled due to an unassigned nodepool status. |
+| RUN-14191 | Fixed an issue where a consolidation failure would cause unnecessary evictions. |
+| RUN-14154 | Fixed an issue where the cluster wizard dropdown listed versions that were incompatible with the installed control plane. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-15.md:54]: "cluster wizard" is actually "new cluster" something (wizard?)

File: docs/home/changelog/hotfixes-2-15.md [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+| RUN-13956 | Fixed an issue where templates were not edited successfully. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-15.md:55]: Jobs templates

**Review by omerbenedictrunai — COMMENTED** [review]
a few comments inline.

File: docs/home/changelog/hotfixes-2-15.md (hunk 10 above)
Comment (jasonnovichRunAI) [inline docs/home/changelog/hotfixes-2-15.md:50]:
```suggestion
| RUN-14292 | Fixed an issue where BCM installations were failing due to missing `create cluster` permissions. |
```

File: docs/home/changelog/hotfixes-2-15.md (hunk 12 above)
Comment (jasonnovichRunAI) [inline docs/home/changelog/hotfixes-2-15.md:55]:
```suggestion
| RUN-13956 | Fixed an issue where the *Jobs* templates were not edited successfully. |
```

File: docs/home/changelog/hotfixes-2-15.md (hunk 11 above)
Comment (jasonnovichRunAI) [inline docs/home/changelog/hotfixes-2-15.md:54]:
```suggestion
| RUN-14154 | Fixed an issue in the *New cluster* form, whefre the dropdown listed versions that were incompatible with the installed control plane. |
```

File: docs/home/changelog/hotfixes-2-13.md (hunk 8 above)
Comment (jasonnovichRunAI) [inline docs/home/changelog/hotfixes-2-13.md:49]:
```suggestion
| RUN-13171 | Fixed an issue when a cluster is not connected the actions in the *Workspace* and *Training* pages are still enabled. After the corrections, the actions will be disabled. |
```

File: docs/home/changelog/hotfixes-2-13.md (hunk 9 above)
Comment (jasonnovichRunAI) [inline docs/home/changelog/hotfixes-2-13.md:90]:
```suggestion
```

**Pull request merged**